app/code/Magento/Inventory/Model/SourceSelectionInterface.php

```diff
@@ -0,0 +1,18 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Inventory\Model;
+
+interface SourceSelectionInterface
+{
+    /**
+     * @param $store
+     * @param $items
+     * @param \Magento\Quote\Model\Quote\Address $destinationAddress
+     * @return Package[]
+     */
+    public function getPackages($store, $items, $destinationAddress);
```

Comment:
why do we accept $store parameter here? 
Just to read the configuration data? Why can't we resolve it? 

in any case if we want introduce scoping , which we usually not do - it should be called $scope , but not $store.